### PR TITLE
Get meshes folder

### PR DESCRIPTION
**Pull request by BenjaminDeleze**
- [x] from Webots upload
- [x] from GUI

### CODE REVIEW

File: httpdocs/js/webots-cloud.js [hunk 1]
```diff
@@ -820,6 +820,17 @@ document.addEventListener('DOMContentLoaded', function() {
             </span>
           </div>
           <div class="help">Upload all the texture files: <em>image.png</em>, <em>image.jpg</em> and <em>image.hdr</em></div>
```
Comment (omichel) [inline httpdocs/js/webots-cloud.js]:
```suggestion
          <div class="help">Upload all the texture files: <em>*.png</em>, <em>*.jpg</em> and/or <em>*.hdr</em></div>
```

File: httpdocs/js/webots-cloud.js [hunk 2]
```diff
@@ -820,6 +820,17 @@ document.addEventListener('DOMContentLoaded', function() {
...
+        <div class="field">
+          <label class="label">Mesh files</label>
+          <div class="control has-icons-left">
+            <input id="texture-files" name="meshes[]" class="input" type="file" multiple accept=".stl, .obj, .mtl, .dae">
+            <span class="icon is-small is-left">
+              <i class="fas fa-upload"></i>
+            </span>
+          </div>
+          <div class="help">Upload all the meshes files: <em>mesh.obj</em>, <em>mesh.mtl</em>,
+            <em>mesh.dae</em> and <em>mesh.stl</em></div>
```
Comment (omichel) [inline httpdocs/js/webots-cloud.js]:
```suggestion
          <div class="help">Upload all the meshes files: <em>*.obj</em>, <em>*.mtl</em>,
            <em>*.dae</em> and/or <em>*.stl</em></div>
```

File: httpdocs/ajax/animation/create.php [hunk 3]
```diff
@@ -140,14 +144,24 @@ function parse_mf_string($line, $parameter) {
...
+    }
+  }
+  if ($total_meshes > 0) {
+    mkdir("$folder/meshes");
+    for($i = 0; $i < $total_meshes; $i++) {
+      $target = basename($_FILES['meshes']['name'][$i]);
+      if ($target == '')
+        continue;
+      if (!move_uploaded_file($_FILES['meshes']['tmp_name'][$i], "$folder/meshes/$target"))
+        error("Cannot move $total_meshes $target");
```
Comment (omichel) [inline httpdocs/ajax/animation/create.php]: Instead of duplicating the code, it would be better to create a new function and call it twice.

**Review by omichel — CHANGES_REQUESTED** [review]

**Comment by BenjaminDeleze** [comment]
I took the opportunity to clean the DB and the different storage folders.

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**